### PR TITLE
Update bin script for launching in production

### PR DESCRIPTION
**Pull request by ccare**
Added a `#!/bin/sh` hashbang line to bin script. 

Also set the `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=true` environment-variable which appears to be necessary.

### CODE REVIEW

**Comment by ccare** [comment]
I don't know really. Without it, I get the error:

```
Couldn't find a valid ICU package installed on the system. Set the configuration flag System.Globalization.Invariant to true if you want to run with no globalization support.
```

I'm assuming that globalization is a as I18n/internationalisation thing. Maybe I don't have a locale or similar setup? Bit weird really.

File: Dockerfile [hunk 1]
```diff
@@ -12,11 +12,11 @@ RUN dotnet publish -c Release -r linux-musl-x64 -o /opt/analyzer
 
 # Create analyze script
 RUN mkdir /opt/analyzer/bin && \
-    echo "/opt/analyzer/Exercism.Analyzers.CSharp \$1 \$2" > /opt/analyzer/bin/analyze.sh && \
+    printf "#!/bin/sh\n\nexport DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=true\n/opt/analyzer/Exercism.Analyzers.CSharp \$1 \$2\n" >/opt/analyzer/bin/analyze.sh && \
```
Comment (ErikSchierboom) [inline Dockerfile]: Did you find a resource that describes how this environment variable helps?

Comment (ccare) [reply Dockerfile]: Only that https://github.com/dotnet/coreclr/issues/14927 identifies this as a way of setting `System.Globalization.Invariant` without needing to play with the dotnet env

Comment (ErikSchierboom) [reply Dockerfile]: I would have expected something like this: ENV COMPlus_EnableDiagnostics=0

See this thread: https://github.com/dotnet/docs/issues/10217

I'd be totally fine with adding that to the Dockerfile.

Comment (ccare) [reply Dockerfile]: I switched to `printf` because `echo` (in that underlying distro, or shell) didn't expand `\n`

**Comment by ErikSchierboom** [comment]
Thanks for doing this!

**Pull request merged**